proxy/http/HttpSessionManager.h

```diff
@@ -67,6 +67,9 @@ class ServerSessionPool : public Continuation
   static bool validate_host_sni(HttpSM *sm, NetVConnection *netvc);
   static bool validate_sni(HttpSM *sm, NetVConnection *netvc);
   static bool validate_cert(HttpSM *sm, NetVConnection *netvc);
+  void removeSession(PoolableSession *ssn);
```

Comment:
Can you make `removeSession` and `addSession` private functions for now because H1ServerSession doesn't need to call them directly?

I know the new functions are used by Http2ServerSession, and we can discuss the detail later on H2-to-Origin PR, but having `removeSession` and `addSession` as public functions in addition to `acquireSession` and `releaseSession` doesn't make sense to me. The interface would be confusing. I think the both H1ServerSession and H2ServerSession should be handled by HttpSessionManager consistently - unnecessary processes should be skipped inside HttpSessionManager.

Reply:
Sure, in this use case they can be made private